apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -112,8 +116,17 @@ const ConfigScreen = () => {
     });
   };
 
+  const clearIsAppConfigureCalled = () => {
+    setIsAppConfigureCalled(false);
+  };
```

Comment:
not super sold on this name. thoughts? This essentially clears the ephemeral state of knowing that the Install/Save button has been clicked.

Reply:
Yeah totally valid assumptions based on the naming. [This](https://github.com/contentful/apps/pull/7299#discussion_r1567855792) comment explains as well but essentially I needed to create logic that allows child components to know when the user has clicked "save/install" and the isAppConfigureCalled is the state telling the components this. The aformentioned child components are specifically the preview path inputs, that when the user saves/installs, and the input is empty, the warning message appears. This allows the user to save/install without inputing those values but still informs the user that its not a functioning value. Where I took some liberty is that this isAppConfigureCalled is an ephemeral "true" once the user saves/installs, and when the user then interacts with the Content type section, this then resets using this `clearIsAppConfigureCalled` state, to the default false. I can certainly think a bit harder about naming these to ensure that this confusion is limited, which I had a feeling would happen 😅 I also think perhaps engaging anywhere on the config page could reset this state, as opposed to the user just clicking in the Content type section.